pkg/populator/populator.go

```diff
@@ -72,7 +73,7 @@ func (p *Populator) handlePVUpdate(pv *v1.PersistentVolume) {
 			if !found {
 				return
 			}
-			if provisioner == p.Name {
+			if provisioner == p.Name || strings.HasPrefix(provisioner, p.Name) {
```

Comment:
may I  use "strings.HasPrefix(provisioner, p.Name) " to replace "provisioner == p.Name "?